codegenerator/cli/templates/static/codegen/src/UserContext.res

```diff
@@ -187,15 +212,28 @@ let makeEntityLoaderContext = (params): Types.entityLoaderContext<
   Entities.internalEntity,
   unknown,
 > => {
+  let get = entityId =>
+    params.loadLayer->LoadLayer.loadById(
+      ~entityConfig=params.entityConfig,
+      ~inMemoryStore=params.inMemoryStore,
+      ~shouldGroup=params.shouldGroup,
+      ~eventItem=params.eventItem,
+      ~entityId,
+    )
   {
-    get: entityId =>
-      params.loadLayer->LoadLayer.loadById(
-        ~entityConfig=params.entityConfig,
-        ~inMemoryStore=params.inMemoryStore,
-        ~shouldGroup=params.shouldGroup,
-        ~eventItem=params.eventItem,
-        ~entityId,
-      ),
+    get,
+    getOrThrow: (entityId, ~message=?) =>
+      get(entityId)->Promise.thenResolve(entity => {
+        switch entity {
+        | Some(entity) => entity
+        | None =>
```

Comment:
How does this implementation account for the entity not existing on the first run of the loader?

Reply:
@JonoPrest I updated the PR to silently ignore the first loader run errors. I don't know why we didn't do it before. I firmly believe that the previous behavior is wrong, so I'll merge the PR without a rereview, but I'm open to feedback when you're back on Monday.

Reply:
Ok, something is wrong with thests, so you'll have a chance to review :)